evergreen.yml

```diff
@@ -15,10 +15,28 @@ functions:
       working_dir: src
       script: |
         set -o errexit
-        virtualenv --python=/opt/mongodbtoolchain/v3/bin/python3 venv
+        set -o verbose
+
+        /opt/mongodbtoolchain/v3/bin/python3 -m venv venv
         . venv/bin/activate
 
-        pip install -r requirements.txt
+        pip install poetry
+        poetry install
+
+  deploy:
```

Comment:
How were we deploying before, out of curiosity?

Reply:
manually running the commands.